doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_keys.rst

```diff
@@ -55,15 +55,15 @@ To generate the keys, follow these steps:
 
    * For the application core::
 
-         python generate_psa_key_attributes.py --usage VERIFY_MESSAGE_EXPORT --id 0x40022100 --type ECC_TWISTED_EDWARDS --size 255 --algorithm EDDSA_PURE --location PERSISTENT_CRACEN --key-from-file MANIFEST_APPLICATION_GEN1_pub.pem  --file all_keys.json --cracen_usage RAW
+        python generate_psa_key_attributes.py --usage VERIFY_MESSAGE_EXPORT --id 0x40022100 --type ECC_TWISTED_EDWARDS --size 255 --algorithm EDDSA_PURE --location LOCATION_CRACEN --key-from-file MANIFEST_APPLICATION_GEN1_pub.pem  --file all_keys.json --cracen_usage RAW --lifetime PERSISTENCE_DEFAULT
 
    * For the radio core::
 
-         python generate_psa_key_attributes.py --usage VERIFY_MESSAGE_EXPORT --id 0x40032100 --type ECC_TWISTED_EDWARDS --size 255 --algorithm EDDSA_PURE --location PERSISTENT_CRACEN --key-from-file MANIFEST_RADIOCORE_GEN1_pub.pem --file all_keys.json --cracen_usage RAW
+        python generate_psa_key_attributes.py --usage VERIFY_MESSAGE_EXPORT --id 0x40032100 --type ECC_TWISTED_EDWARDS --size 255 --algorithm EDDSA_PURE --location LOCATION_CRACEN --key-from-file MANIFEST_RADIOCORE_GEN1_pub.pem --file all_keys.json --cracen_usage RAW --lifetime PERSISTENCE_DEFAULT
 
    * For the main root manifest::
 
-         python generate_psa_key_attributes.py --usage VERIFY_MESSAGE_EXPORT --id 0x4000AA00 --type ECC_TWISTED_EDWARDS --size 255 --algorithm EDDSA_PURE --location PERSISTENT_CRACEN --key-from-file MANIFEST_OEM_ROOT_GEN1_pub.pem --file all_keys.json --cracen_usage RAW
+        python generate_psa_key_attributes.py --usage VERIFY_MESSAGE_EXPORT --id 0x4000AA00 --type ECC_TWISTED_EDWARDS --size 255 --algorithm EDDSA_PURE --location LOCATION_CRACEN --key-from-file MANIFEST_OEM_ROOT_GEN1_pub.pem --file all_keys.json --cracen_usage RAW --lifetime PERSISTENCE_DEFAULT
```

Comment:
Line 81 and below:

* The ``metadata`` field from the JSON file is used for the function's attributes argumen**t.** (full stop missing)
* The ``value`` field is passed to the function's data argumen**t.**(full stop missing)
* The function's ``data_length`` **parameter** is set to the length of the value field.